cardigan_new/stories/components/ArchiveTree/ArchiveTree.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import collectionTree from '@weco/cardigan_new/stories/data/collection-tree';
+import ArchiveTree from '@weco/content/components/ArchiveTree';
+import IsArchiveContext from '@weco/content/components/IsArchiveContext/IsArchiveContext';
+
+const meta: Meta<typeof ArchiveTree> = {
+  title: 'Components/ArchiveTree',
+  component: ArchiveTree,
+  args: {
+    work: collectionTree,
+  },
+  parameters: {
+    disableSnapshot: true,
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof ArchiveTree>;
+
+const Template = args => (
+  <IsArchiveContext.Provider value={true}>
+    <ArchiveTree {...args} />
```

Comment:
Out of scope to fix this, already that it's displaying something is great :D 

Could we amend this ticket though based on what gets updated here? 🙏  https://github.com/wellcomecollection/wellcomecollection.org/issues/9157

<img width="348" alt="Screenshot 2024-10-15 at 17 30 23" src="https://github.com/user-attachments/assets/669694f8-c3d7-4bae-90fd-f4691f7941ec">

Reply:
Yeah, I think I'd demo-ed it to you with one more item displaying, but that didn't seem  like enough of an improvement to warrant the addition of the `addon-mock` that I'd included (and apparently couldn't be hidden on other stories), so I took it out again. Wrapping it in the correct context was what made _something_ render. I agree that looking into getting more of a tree displaying belongs to a different ticket. I'll update #9157